tests/test_person.py

```diff
@@ -1,57 +1,57 @@
 import datetime
 
-from django.core.urlresolvers import reverse
+from django.urls import reverse, resolve
 import pytest
 
 from .conftest import get_uid_chunk
-from lametro.views import LAMetroPerson
+from lametro.views import PersonDetailView, LAPersonDetailView
 
 
 @pytest.mark.django_db
-def test_person_page_redirects(client, metro_person, mocker):
+def test_person_page_redirects(client, metro_person, mocker, rf):
```

Comment:
What is rf?

Reply:
https://pytest-django.readthedocs.io/en/latest/helpers.html#rf-requestfactory

do you know of a way to make this more explicit?

Reply:
ha, but I don't actually use it. removing.